internal/pkg/externalplugins/lgtm/lgtm.go

```diff
@@ -46,37 +39,18 @@ var (
 
 // HelpProvider constructs the PluginHelp for this plugin that takes into account enabled repositories.
 // HelpProvider defines the type for function that construct the PluginHelp for plugins.
-func HelpProvider(epa *tiexternalplugins.ConfigAgent) externalplugins.ExternalPluginHelpProvider {
+func HelpProvider(_ *tiexternalplugins.ConfigAgent) externalplugins.ExternalPluginHelpProvider {
 	return func(enabledRepos []config.OrgRepo) (*pluginhelp.PluginHelp, error) {
-		configInfo := map[string]string{}
-		cfg := epa.Config()
-		for _, repo := range enabledRepos {
-			opts := cfg.LgtmFor(repo.Org, repo.Repo)
-			var isConfigured bool
-			var configInfoStrings []string
-			configInfoStrings = append(configInfoStrings, "The plugin has these configurations:<ul>")
-			if opts.ReviewActsAsLgtm {
-				configInfoStrings = append(configInfoStrings, "<li>"+configInfoReviewActsAsLgtm+"</li>")
-				isConfigured = true
-			}
-			configInfoStrings = append(configInfoStrings, "</ul>")
-			if isConfigured {
-				configInfo[repo.String()] = strings.Join(configInfoStrings, "\n")
-			}
-		}
 		pluginHelp := &pluginhelp.PluginHelp{
 			Description: "The ti-community-lgtm plugin manages the 'status/LGT{number}' (Looks Good To Me) label.",
-			Config:      configInfo,
 		}
 
 		pluginHelp.AddCommand(pluginhelp.Command{
-			Usage:       "/lgtm [cancel] or triggers by GitHub review action.",
-			Description: "Add or remove the 'status/LGT{number}' label. Additionally, the PR author can use '/lgtm cancel'.",
+			Usage:       "Triggered by GitHub review action.",
```

Comment:
We may need to elaborate here, only approve and request changes have corresponding effects?

Reply:
OK. Updating...